mtp_cashbook/templates/cashbook/transaction_batch_list.html

```diff
@@ -55,19 +57,9 @@
         </tr>
       {% endfor %}
       </tbody>
-
-      <tfoot>
-        <tr>
-          <th colspan="3" class="print">
-            <a href="#print-dialog" class="js-Dialog">{% trans "Print batch" %}</a>
-          </th>
-          <th colspan="2" class="select-all">
-            <input type="checkbox" id="select-all" class="Checkbox--right">
-            <label for="select-all">{% trans "Select all" %}</label>
-          </th>
-        </tr>
-      </tfoot>
     </table>
+
+    {% include "cashbook/includes/table_actions.html" with location='footer' field_name=form.transactions.html_name %}
```

Comment:
Yep, they are linked to the contents of the table so would make more sense within `thead` and `tfoot`.